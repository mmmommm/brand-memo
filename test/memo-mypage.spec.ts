import MemoMypage from '@/views/memo-mypage.vue'
import Vuetify from 'vuetify'

import { createLocalVue, shallowMount } from "@vue/test-utils"

describe('memo-mypage.vue', () => {
  const localVue = createLocalVue()
  let vuetify

  beforeEach(() => {
    vuetify = new Vuetify()
  })
  const wrapper = shallowMount(MemoMypage, {
    localVue,
    vuetify,
    data() {
      return {
        isLoading: false,
        haveMemo: false,
        memos: [
          {
            code: 1111,
            slug: "11111111",
            name: "dummy",
            capitalization: 1,
            floating: 1,
            theme: "dummy",
            price: 1,
            url: "https://dummy.ac.jp",
            reason: "dummy",
          },
        ],
      }
    }
  })
  it('<MemoMypage />', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})