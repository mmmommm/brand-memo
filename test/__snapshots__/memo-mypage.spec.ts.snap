// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`memo-mypage.vue <MemoMypage /> 1`] = `
<v-app-stub
  id="app"
>
  <loadingscreen-stub
    style="display: none;"
  />
   
  <layout-stub>
    <!---->
     
    <v-container-stub
      tag="div"
    >
      <v-layout-stub
        align-center=""
        column=""
        fill-height=""
        justify-center=""
        tag="div"
      >
        <h1
          class="display-2 font-weight-thin my-12"
        >
          
            まだメモがありません
          
        </h1>
         
        <h4
          class="subheading mt-12"
        >
          
            是非下記のボタンを押下しメモを作成してみてください。
          
        </h4>
         
        <v-btn-stub
          activeclass=""
          class="mt-4"
          outlined="true"
          rounded="true"
          tag="button"
          to="/MemoAdd"
          type="button"
        >
          
            メモを作成する
          
        </v-btn-stub>
      </v-layout-stub>
    </v-container-stub>
  </layout-stub>
</v-app-stub>
`;
